content/en/observability_pipelines/processors/remap_ocsf/custom_configuration.md

```diff
@@ -0,0 +1,418 @@
+  "metadata": {
+    "version": "1.1.0",
+    "class": "Network Activity",
+    "profiles": ["container", "host", "security_control"]
+  },
+  "preprocess": [],
+  "mapping": []
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+The mapping descriptor has four main sections:
+
+| Name       | Required  | Description                                                                                                           |
+|------------|-----------|-----------------------------------------------------------------------------------------------------------------------|
+| `version`    | Yes       | Must be set to `1` to indicate the mapping descriptor format version.                                                 |
+| `metadata`   | Yes       | Contains a set of hard-coded description fields about the event class. See [Metadata section](#metadata-section) for details.                         |
+| `preprocess` | No        | An ordered list of preprocessing steps. The preprocessors reformat raw data from the sources so that the data can be converted to OCSF format based on `mapping`. Each preprocessor entry consists of a `function` and parameters associated with that function. See [Preprocessors](#preprocess-section) for more information. |
+| `mapping`    | Yes       | An ordered list of field-to-field assignments, where a `source` field is assigned to a `dest` field in the output OCSF event. See  for more information. Each [mapping](#mapping-section) may have a `conversion` specified by a [lookup table](#mapping-lookup-tables) or post-processing [mapping function](#mapping-functions). |
+
+### Metadata section
+
+The metadata section requires the following:
+
+| Name     | Required | Description                                                                                     |
+|----------|----------|-------------------------------------------------------------------------------------------------|
+| `version`  | Yes      | The target OCSF schema version.                                                                 |
+| `class`    | Yes      | The name of the OCSF event class. The event class ID, category name, and category ID are derived from this. |
+| `profiles` | Yes      | An array of profile names that are added to the event class schema.                             |
+
+### Preprocess section
+
+The `preprocess` section lists the preprocessors that reformats the data to allow field-to-field mappings. Each entry in this section consists of a `function` and the parameters associated with that function.
+
+#### `parse_csv`
+
+The `parse_csv` preprocessor:
+
+1. Extracts data in the `source` field.
+1. Parses `source` as CSV and maps the values to the `columns` listed.
+1. Inserts the mapped data in the `dest` field.
+
+Columns with a `null` value are dropped. One of the columns can have a wildcard (`*`), in which case it is assigned a string containing all the text from the fields that remain after those before and after have been mapped.
```

Comment:
Need clarification on what this means: "assigned a string containing all the text from the fields that remain after those before and after have been mapped."

Reply:
@bruceg could you clarify that part of the sentence?

Reply:
See the example posted earlier. Effectively, the output column is assigned everything that is left after handling the remainder of the fields after the wildcard.

Reply:
> in which case it is assigned a string containing all the text from the fields that remain after those before and after have been mapped.

@bruceg is this modification to that part of the sentence correct?

> where that column is assigned a string containing all of the remaining field values that have not mapped at the end of the mapping process.

Reply:
It is effectively correct, yes. (The actual order of operations is currently different, since the number of fields is known, but it has that effect)